src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -244,7 +244,10 @@ export function ChangeDataView({
                         {
                           defaultMessage:
                             "Switching data views removes the current {textBasedLanguage} query. Save this search to ensure you don't lose work.",
-                          values: { textBasedLanguage },
+                          values: {
+                            textBasedLanguage:
+                              getLanguageDisplayName(textBasedLanguage).toUpperCase(),
+                          },
```

Comment:
Can we include `.toUpperCase()` into `getLanguageDisplayName`?

Reply:
Sure, done here https://github.com/elastic/kibana/pull/168078/commits/4ffc7eacb974d843ca1d15d1ae6c5edae90eb601